From 5b35eac60ac59b0b907a2c16f90163a383a202b7 Mon Sep 17 00:00:00 2001
From: Vernon Yang <vernon2gm@gmail.com>
Date: Tue, 20 Dec 2022 22:26:04 +0800
Subject: [PATCH 23/25] maple_tree: change return type of mas_commit_b_node()

The return value of mas_commit_b_node() function represents whether
the submit b_node is successful, and can only be false and true, so
the return value type is bool is more appropriate and for better
readability.

Signed-off-by: Vernon Yang <vernon2gm@gmail.com>
---
 lib/maple_tree.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/lib/maple_tree.c b/lib/maple_tree.c
index 1638925e2..12891c606 100644
--- a/lib/maple_tree.c
+++ b/lib/maple_tree.c
@@ -3581,7 +3581,7 @@ static inline bool mas_reuse_node(struct ma_wr_state *wr_mas,
  * @b_node: The maple big node
  * @end: The end of the data.
  */
-static inline int mas_commit_b_node(struct ma_wr_state *wr_mas,
+static inline bool mas_commit_b_node(struct ma_wr_state *wr_mas,
 			    struct maple_big_node *b_node, unsigned char end)
 {
 	struct maple_node *node;
@@ -3601,7 +3601,7 @@ static inline int mas_commit_b_node(struct ma_wr_state *wr_mas,
 
 	mas_node_count(wr_mas->mas, 1);
 	if (mas_is_err(wr_mas->mas))
-		return 0;
+		return false;
 
 	node = mas_pop_node(wr_mas->mas);
 	node->parent = mas_mn(wr_mas->mas)->parent;
@@ -3610,7 +3610,7 @@ static inline int mas_commit_b_node(struct ma_wr_state *wr_mas,
 	mas_replace(wr_mas->mas, false);
 reuse_node:
 	mas_update_gap(wr_mas->mas);
-	return 1;
+	return true;
 }
 
 /*
-- 
2.39.0.rc2.1.gbd5df96b79

